include/odp/api/spec/crypto_types.h

```diff
@@ -897,27 +897,29 @@ typedef enum {
 typedef enum {
 	/** Algorithm successful */
 	ODP_CRYPTO_ALG_ERR_NONE,
-	/** Invalid data block size */
+	/** Invalid range or input or output packet size */
```

Comment:
If there is no reason to explicitly mention input/output packet, this could be simply:
"Invalid data range or packet size"

op_param_t use term "data range", so I'd use that also here.

```
	/** Data range to apply cipher */
	odp_packet_data_range_t cipher_range;

```

Reply:
changed in v5